packages/vite/src/node/constants.ts

```diff
@@ -18,7 +18,7 @@ export const ESBUILD_MODULES_TARGET = [
   'edge88',
   'firefox78',
```

Comment:
Did you mean this?
(+ undoing the other change)
```suggestion
  'firefox71', // transpile nullish coalescing
```

Reply:
I mean reverting to safari 13.1, as you said this issue wasn't present in v2.9 + es2020. But I think safari 14 is needed